resources/popup.js

```diff
@@ -1,27 +1,29 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
```

Comment:
Why did you put this method in the global namespace instead of inside the function scope?